styles.css

```diff
@@ -0,0 +1,406 @@
+*{
+  margin: 0;
+  padding:0;
+}
+body{
+  background-size:cover;
+}
+@media print{
+  body img{
+    display:none;
+  }
+}
+a{
+  text-decoration: none;
+}
+/* Mobile Styles */
+@media only screen and (max-width: 500px){
+  .container{
+    margin: 30px 50px;
+    display: grid;
+    grid-gap: 15px;
+  }
+  header{
+    width:100% !important;
+    background-image:url("images/header-image.jpg"),linear-gradient(to right,rgba(255,0,0,0),rgba(255,0,0,1));
+    background-size:cover !important;
+  }
+  h4{
+    padding: 2px !important;
+  }
+  nav{
+    width:100% !important;
+    height:25px !important;
+  }
+  .main-container{
+   margin: 3px !important;
+   grid-gap:5px !important;
+  }
+  .intro{
+   width:50%;
+  }
+  .sidebar img,p{
+    display: none;
+  }
+  section{
+    display:flex;
+    flex-wrap: wrap !important;
+    width:100% !important;
+  }
+  .section-3{
+    margin: 20px;
+  }
+  .section-left-side{
+    flex: 0% !important;
+    margin:2px;
+  }
+  .section-right-side{
+    flex: 50% !important;
+    flex-wrap: wrap !important;
+    padding: 1px !important;
+  }
+  .section-right-side h3{
+    font-size: medium;
+  }
+  p{
+    padding: 0px !important;
+    font-size: 0.7em !important;
+    word-wrap: break-word !important;
+  }
+  article{
+    grid-template-columns: auto !important;
+    grid-gap:2px !important;
+    width: 100% !important;
+  }
+  .row-1 {
+    display: grid;
+    grid-template-columns: auto !important;
+    grid-gap: 10px;
+    }
+  .col{
+    width: 50%;
+  }
+  .card-desc{
+    padding:2px !important;
+  }
+  .card-desc p{
+    padding: 0px !important;
+    font-size: 0.8 !important;
+  }
+  footer{
+    width:100% !important;
+  }
+}
+/* Tablet Styles */
+@media only screen and (min-width:501px) and (max-width:900px){
+  .container{
+    margin: 30px 50px;
+    display: grid;
+    grid-gap: 15px;
+    grid-template-columns: auto auto;
+  }
+  h4{
+    padding: 2px !important;
+  }
+  nav{
+    width: 100% !important;
+    height: 25px !important;
+  }
+  body{
+    width: 100%;
+  }
+  .main-container{
+    margin: 3px !important;
+    grid-gap:5px !important;
```

Comment:
Again do not use px. The mockups for this assignment have gutters on either side of the page content. You should utilize some whitespace to either side of your content by setting the left and right margins to something like 10%